include/wally_elements.h

```diff
@@ -162,6 +210,46 @@ WALLY_CORE_API int wally_asset_rangeproof(
     size_t len,
     size_t *written);
 
+/**
+ * Generate an explicit value rangeproof for a transaction output.
+ *
+ * The nonce for this function should be randomly generated.
+ * See `wally_asset_rangeproof_with_nonce`.
+ */
+WALLY_CORE_API int wally_explicit_rangeproof(
+    uint64_t value,
+    const unsigned char *nonce,
+    size_t nonce_len,
+    const unsigned char *vbf,
+    size_t vbf_len,
+    const unsigned char *commitment,
```

Comment:
Why is `commitment` needed? can't we compute it from `value` and `vbf`?

Reply:
2 reasons:

(1) consistency with the arguments to the non-explicit call.

(2) In common usage you need to compute the commitment for other pruposes anyway, e.g. in PSET we set the commitment, full range proof and the explicit value proof at the same time. computing it 3 times is wasteful.